Makefile

```diff
@@ -1,18 +1,23 @@
 .PHONY: start
 
 # Read values as needed from .env
+#
+# Optionally pass 'environment=<something>' to load env vars from '.env-something' instead
+# Useful to separate i.e. dev from testing databases
+#
 # If using the same variables in recipes that need to use a dotenv file other
 # than .env, remember to check that no values from .env are being used
 # inadvertently.
-ENVFILE := $(if $(environment), .env-test-e2e, .env)
+ENVFILE := $(if $(environment), .env-$(environment), .env)
+ifneq (,$(wildcard $(ENVFILE)))
+    include $(ENVFILE)
+    export
+endif
+
 CIENV := $(if $(filter $(environment), ci), -f docker-compose-test-e2e.ci.yml , -f docker-compose-test-e2e.local.yml)
 API_DB_INSTANCE := $(if $(environment), test-e2e-postgresql-api, postgresql-api)
 GEO_DB_INSTANCE := $(if $(environment), test-e2e-postgresql-geo-api, postgresql-geo-api)
 REDIS_INSTANCE := $(if $(environment), test-e2e-redis, redis)
-_API_POSTGRES_USER := $(if $(filter $(environment), ci),${API_POSTGRES_USER},$(shell grep -e API_POSTGRES_USER ${ENVFILE} | sed 's/^.*=//'))
-_API_POSTGRES_DB := $(if $(filter $(environment), ci),${API_POSTGRES_DB},$(shell grep -e API_POSTGRES_DB ${ENVFILE} | sed 's/^.*=//'))
-_GEO_POSTGRES_USER := $(if $(filter $(environment), ci),${GEO_POSTGRES_USER},$(shell grep -e GEO_POSTGRES_USER ${ENVFILE} | sed 's/^.*=//'))
-_GEO_POSTGRES_DB := $(if $(filter $(environment), ci),${GEO_POSTGRES_DB},$(shell grep -e GEO_POSTGRES_DB ${ENVFILE} | sed 's/^.*=//'))
```

Comment:
it's been a looong time (and I barely remember details from last week, if I didn't document them), but IIRC the reason for using a prefix on these make vars was to avoid clashes with env vars (from which they *may* be set, or not, depending on the environment), so *I think* this change will break CI or running things in Docker locally.

Reply:
shame on me for not clearly documenting/commenting why the dangling underscore, of course

also - `_something` may instinctively trigger "unused var/placeholder var" instinct, so the choice may not be the best. I have a vague recollection of not being able to decide on a better naming schema to differentiate the two layers of variables and ending up using a quick change to get the task done.